app/src/main/java/io/intrepid/contest/base/BaseActivity.java

```diff
@@ -92,4 +92,8 @@ protected void setActionBarTitle(String title) {
             getSupportActionBar().setTitle(title);
         }
     }
+
+    protected void setActionBarTitle(@StringRes int titleResource) {
+        setActionBarTitle(getResources().getString(titleResource));
```

Comment:
I agree with @jmccormack200 we should overload the setActionbarTitle method to accept int res.